src/pages/Projects/ProjectsGallery.jsx

```diff
@@ -1,29 +1,41 @@
 import React from "react";
-import CardProject from "./CardProject";
 import chalets from "./Chalet";
 import roads from "./Roads";
 import reunions from "./Reunions";
-import mamastes from "./Mamaste";
-
-function createCard(project) {
-  return <CardProject name={project.name} imgURL={project.imgURL} />;
-}
+import Carousel from "../../components/carousel/Carousel";
 
 function ProjectsGallery() {
+  const simpleCarouselSettings = {
+    dots: true,
+    infinite: true,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    autoplay: true,
+    speed: 7000,
+    autoplaySpeed: 2000,
+    cssEase: "linear",
+  };
+
   return (
     <div className="projects-gallery">
-      <p>
-        <a href="https://www.instagram.com/chalet_designinteriores/">
-          Chalet - Interior Design
-        </a>
-      </p>
-      {chalets.map((chalet) => createCard(chalet))}
-      <p>Estrada Fora - Travel Blog</p>
-      {roads.map((road) => createCard(road))}
-      <p>Mamaste - Lactation Consultations</p>
-      {mamastes.map((mamaste) => createCard(mamaste))}
-      <p>Reunion - Le Wagon Bootcamp's final project</p>
-      {reunions.map((reunion) => createCard(reunion))}
+      <div>
+        <p>
+          <a href="https://www.instagram.com/chalet_designinteriores/">
```

Comment:
why is this not in s3?

Reply:
This is not an image, it is a link to instagram page